votor/src/certificate_pool/parent_ready_tracker.rs

```diff
@@ -84,8 +84,8 @@ impl ParentReadyTracker {
         }
     }
 
-    /// Adds a new notar-fallback certificate
-    pub fn add_new_notar_fallback(
+    /// Adds a new notarize fallback certificate, we can use Notarize/NotarizeFallback/FastFinalize
+    pub fn add_new_notar_fallback_or_equivalent(
```

Comment:
nit: I would call this `add_new_notar_fallback_or_stronger`

Reply:
Done.